pkg/kubectl/cmd/wait/wait.go

```diff
@@ -389,6 +404,9 @@ func (w ConditionalWait) checkCondition(obj *unstructured.Unstructured) (bool, e
 }
 
 func (w ConditionalWait) isConditionMet(event watch.Event) (bool, error) {
+	if event.Type == watch.Error {
+		return true, nil
```

Comment:
This could be 
```
if event.Type == watch.Error {
	err = apierrors.FromObject(event.Object)
	fmt.Fprintf(o.ErrOut, "error: An error occurred while waiting for the condition to be satisfied: %v", err)
	// this will chain back out, result in another get and an return false back up the chain
	return false, nil
}
```
And the condition check would appear more normal.

Same for above

Reply:
conditional wait doesn't have `ErrOut`, I was considering whether to do this or not.  I'm fine with it